calculator_v5.rb

```diff
@@ -0,0 +1,93 @@
+# Goal is to create a program that takes two numbers and performs mathematical operations on them, then gives the result. Shooting for Gold level here.
+
+# outputs welcome string and user prompt
+puts "Welcome to Mathtropolis! Let's do some math!"
+puts "What type of math would you like to do?"
+
+# assigns user input to the variable "operation"
+operation = gets.chomp.downcase
+
+# defines a method called do_math that takes 3 arguments corresponding to
+# operation and inputted numbers below
+def do_math(operation, num1, num2)
```

Comment:
Very good to isolate functionality into reusable methods. ++ for that. It threw me for a moment that we started doing stuff (`puts` on lines 4 & 5, then getting `operation`), then switched to method definition.

Since this is a pretty long method, I'd recommend having it (along with other methods, if any) at the very beginning of the file. Do all the `def`s, then have the code that utilizes the methods.